### PR TITLE
2016-02-24-support-plan-of-ruby-2-0-0-and-2-1 (ru)

### PR DESCRIPTION
**Pull request by razum2um**
/cc @gazay @arbox

### CODE REVIEW

**Comment by stomar** [comment]
Please wrap at 80 characters.

**Comment by razum2um** [comment]
@stomar :up:

**Comment by gazay** [comment]
good for me. What do you think @arbox?

**Comment by stomar** [comment]
IMO more review not necessary.

File: ru/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Мы очень рекомендуем вам обновиться на Ruby 2.3 или 2.2.
+
+Пожалуйста, напишите в список рассылки, если вы по каким-либо причинам
+не можете обновиться и желаете продолжать поддерживать эту ветку.
+
+## Ruby 2.1
+
+Версия Ruby 2.1.9 выйдет в конце марта. После этого мы закончим развитие 2.1
+и перейдем в фазу обновлений безопасности. Также, после выхода 2.1.9 мы
```
Comment (arbox) [inline ru/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: No comma after "также".

File: ru/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Подумайте о переходе на Ruby 2.3 или 2.2.
+
+Кстати, мы планируем выпустить Ruby 2.1.10 сразу после 2.1.9.
+Это технический релиз без исправлений, чтобы протестировать выпуск версии
+с двухзначным номером. Этого еще не случалось, поэтому мы думаем,
+что необходимо это попробовать без значительных изменений.
+
+Ruby 2.1.10 не будет отличаться от 2.1.9 ничем, кроме номера версии.
+Вам не обязательно обновляться, но стоит попробовать перед выходом 2.1.11
```
Comment (arbox) [inline ru/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: "необязательно"

File: ru/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Версия Ruby 2.1.9 выйдет в конце марта. После этого мы закончим развитие 2.1
+и перейдем в фазу обновлений безопасности. Также, после выхода 2.1.9 мы
+не будем переносить исправления, не касающиеся безопасности, из новых версий.
+
+Подумайте о переходе на Ruby 2.3 или 2.2.
+
+Кстати, мы планируем выпустить Ruby 2.1.10 сразу после 2.1.9.
+Это технический релиз без исправлений, чтобы протестировать выпуск версии
+с двухзначным номером. Этого еще не случалось, поэтому мы думаем,
```
Comment (arbox) [inline ru/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md]: "двузначным" as for me

**Comment by arbox** [comment]
Fine with me except for small issues.

**Comment by stomar** [comment]
Thanks @arbox, didn't expect such fast feedback... I'll apply those.

**Comment by stomar** [comment]
@razum2um Thanks!

BTW. you should use a feature branch, not `master`.

**Pull request merged**